main.py

```diff
@@ -1,10 +1,43 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    data = json.load(open("players.json", "r")).items()
```

Comment:
Please, use context manager here to work with file with an implementation tips from the checklist